### PR TITLE
Add TSharp

### PR DESCRIPTION
**Pull request by bichanna**
## Adding a language

- [x] The code displays "Hello World"
- [x] I have updated the readme to include the new language
- [x] I have incremented the language count in the readme

[Here's link to the language repo](https://github.com/Tsharp-lang/Tsharp)

### CODE REVIEW

**Review by MrBrain295 — COMMENTED** [review]
looks good I just need a link to the programming language.

**Comment by bichanna** [comment]
Here's the [link](https://github.com/Tsharp-lang/Tsharp) on github.

**Review by MrBrain295 — APPROVED** [review]

File: T Sharp.t# [hunk 1]
```diff
@@ -0,0 +1 @@
+push "Hello World" print
```
Comment (Richienb) [inline T Sharp.t#:1]: @MrBrain295 Next time, check for files not in the right place, like this one.

Comment (MrBrain295) [reply T Sharp.t#:1]: oh, sorry

**Pull request merged**